hugin/gdocs_updater.py

```diff
@@ -99,7 +100,7 @@ def reshape_run_info(self, runs, skiplist=[]):
                 p = [p]
             for project in p:
                 if len(project) == 0:
-                    project = 'Unknown, please check!'
+                    continue
```

Comment:
this is the only effective change right?

Do you know when it can happen than a run does not belong to any project?